app/src/main.cpp

```diff
@@ -828,15 +820,14 @@ namespace scitt
       make_endpoint(
         get_version_path,
         HTTP_GET,
-        tracing_adapter(
-          error_adapter(ccf::json_adapter(get_version)), get_time),
+        ccf::json_adapter(get_version),
         no_authn_policy)
         .set_auto_schema<void, GetVersion::Out>()
         .set_forwarding_required(ccf::endpoints::ForwardingRequired::Never)
         .install();
 
 #ifdef ENABLE_PREFIX_TREE
-      // TODO add tracing_adapter() to all endpoints
+      // TODO local variant of make_endpoint in all endpoints
```

Comment:
So I think this might "just work", in that the prefix tree code calls `registry.make_endpoint`, so it should pick up the new implementation.

On the other hand, it also calls `make_command_endpoint` and `make_read_only_endpoint`, which are what would need to be added here.